services/algolia-sync/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@base-cms/algolia-sync",
+  "version": "1.0.0",
+  "main": "src/index.js",
+  "author": "Jeremy Laird <jlaird@endeavorb2b.com>",
+  "repository": "https://github.com/base-cms/base-cms/tree/master/services/algolia-sync",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "dev": "node src/index.js",
```

Comment:
```suggestion
```
You need to define the entrypoint within the `docker-compose.yml`, and ideally this should be using `gulp` as all other BaseCMS services do. Your development service definition should include an env config for a non-production SQS queue that a developer can use to test this process outside of using production queues.